api/src/test/java/io/strimzi/api/kafka/model/AbstractCrdIT.java

```diff
@@ -7,16 +7,21 @@
 import io.fabric8.kubernetes.client.CustomResource;
 import io.fabric8.kubernetes.client.DefaultKubernetesClient;
 import io.fabric8.kubernetes.client.VersionInfo;
-import io.strimzi.test.BaseITST;
 import io.strimzi.test.TestUtils;
+import io.strimzi.test.k8s.KubeClusterResource;
 import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.TestInstance;
 
+import static io.strimzi.test.k8s.KubeClusterResource.cmdKubeClient;
 import static org.hamcrest.CoreMatchers.is;
 import static org.hamcrest.CoreMatchers.notNullValue;
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.junit.jupiter.api.Assumptions.assumeTrue;
 
-public abstract class AbstractCrdIT extends BaseITST {
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public abstract class AbstractCrdIT {
+
+    protected KubeClusterResource cluster = KubeClusterResource.getInstance();
```

Comment:
the `AbstractResourceOperatorIT` class uses a @BeforeAll annotated method for getting this instance, why it's different here?

Reply:
It does not matter actually. I can rewrite it that on initialize phase.

Reply:
Or @Frawless can do it in following PR with refactor of resources :)

Reply:
I know that's the same, I was just wondering about the different approach and if it had any rationale for that.